### PR TITLE
Feature/479799 edit contact

### PR DESCRIPTION
**Pull request by RaulRodrigo06**

### CODE REVIEW

File: lib/src/widgets/fields/ds_phone_input.widget.dart [hunk 1]
```diff
@@ -1,24 +1,20 @@
+import 'package:blip_ds/blip_ds.dart';
```
Comment (githubdoandre) [inline lib/src/widgets/fields/ds_phone_input.widget.dart:1]: Dont't use the package import, use individual file imports

File: lib/src/widgets/fields/ds_phone_input.widget.dart [hunk 2]
```diff
@@ -56,120 +63,152 @@ class _DSPhoneInputState extends State<DSPhoneInput> {
...
       },
-      child: Column(
-        children: [
-          Container(
-            decoration: BoxDecoration(
-                border: Border.all(
-                  width: 1.0,
-                  color: DSColors.neutralMediumWave,
+      child: Container(
+        padding: const EdgeInsets.fromLTRB(12, 0, 4, 0),
```
Comment (PaollaLira) [inline lib/src/widgets/fields/ds_phone_input.widget.dart:79]: To represent a double, we use the following writing model: 12.0, 0.0, 4.0, 0.0

File: lib/src/widgets/fields/ds_phone_input.widget.dart [hunk 3]
```diff
@@ -56,120 +63,152 @@ class _DSPhoneInputState extends State<DSPhoneInput> {
...
+        decoration: BoxDecoration(
+          color: DSColors.neutralLightSnow,
+          border: Border.all(color: _color()),
+          borderRadius: BorderRadius.circular(8),
```
Comment (PaollaLira) [inline lib/src/widgets/fields/ds_phone_input.widget.dart:83]: To represent a double, we use the following writing model: 8.0

File: lib/src/widgets/fields/ds_phone_input.widget.dart [hunk 4]
```diff
@@ -56,120 +63,152 @@ class _DSPhoneInputState extends State<DSPhoneInput> {
...
+        ),
+        child: Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          mainAxisSize: MainAxisSize.min,
+          children: [
+            Visibility(
+              visible: widget.shouldShowHint && _focusNode.hasFocus,
+              child: Padding(
+                padding: const EdgeInsets.only(top: 8, bottom: 0),
```
Comment (PaollaLira) [inline lib/src/widgets/fields/ds_phone_input.widget.dart:92]: To represent a double, we use the following writing model: 8.0, 0.0

File: lib/src/widgets/fields/ds_phone_input.widget.dart [hunk 5]
```diff
@@ -56,120 +63,152 @@ class _DSPhoneInputState extends State<DSPhoneInput> {
...
+                child: DSText(
+                  widget.hintText,
+                  style: const DSTextStyle(
+                    fontSize: 8,
```
Comment (PaollaLira) [inline lib/src/widgets/fields/ds_phone_input.widget.dart:96]: To represent a double, we use the following writing model: 8.0

File: lib/src/widgets/fields/ds_phone_input.widget.dart [hunk 6]
```diff
@@ -56,120 +63,152 @@ class _DSPhoneInputState extends State<DSPhoneInput> {
...
+                    fontWeight: DSFontWeights.bold,
+                    color: DSColors.neutralMediumCloud,
+                  ),
+                  height: 0,
```
Comment (PaollaLira) [inline lib/src/widgets/fields/ds_phone_input.widget.dart:100]: To represent a double, we use the following writing model: 0.0

File: lib/src/widgets/fields/ds_phone_input.widget.dart [hunk 7]
```diff
@@ -56,120 +63,152 @@ class _DSPhoneInputState extends State<DSPhoneInput> {
...
-                  ),
-                ],
+            if (widget.errorText != null)
+              Padding(
+                padding: const EdgeInsets.only(top: 4.0),
+                child: Row(
+                  children: [
+                    const Icon(DSIcons.error_solid,
+                        color: DSColors.extendRedsDelete),
+                    const SizedBox(width: 6),
```
Comment (PaollaLira) [inline lib/src/widgets/fields/ds_phone_input.widget.dart:202]: To represent a double, we use the following writing model: 6.0

File: lib/src/widgets/fields/ds_text_form_field.widget.dart [hunk 8]
```diff
@@ -24,68 +27,75 @@ class DSTextFormField extends StatefulWidget {
...
-            controller: widget.controller,
-            onChanged: widget.onChanged,
-            style: const DSBodyTextStyle(color: DSColors.neutralDarkCity),
-            autofocus: false,
-            enabled: widget.isEnabled,
-            decoration: InputDecoration(
-              fillColor: widget.isEnabled
+        Obx(
+          () => Container(
+            padding: const EdgeInsets.fromLTRB(12, 6, 4, 6),
```
Comment (PaollaLira) [inline lib/src/widgets/fields/ds_text_form_field.widget.dart:59]: To represent a double, we use the following writing model: 12.0, 6.0, 4.0, 6.0

File: lib/src/widgets/fields/ds_text_form_field.widget.dart [hunk 9]
```diff
@@ -24,68 +27,75 @@ class DSTextFormField extends StatefulWidget {
...
                   ? DSColors.neutralLightSnow
                   : DSColors.neutralLightWhisper,
-              contentPadding: EdgeInsets.zero,
-              border: InputBorder.none,
-              labelText: widget.hintText,
-              labelStyle: const DSCaptionSmallTextStyle(
-                fontWeight: DSFontWeights.bold,
-                color: DSColors.neutralMediumCloud,
+              border: Border.all(color: _borderColor.value),
+              borderRadius: BorderRadius.circular(8),
```
Comment (PaollaLira) [inline lib/src/widgets/fields/ds_text_form_field.widget.dart:65]: To represent a double, we use the following writing model: 8.0

**Review by leonardogbr — APPROVED** [review]

**Review by PaollaLira — APPROVED** [review]

**Pull request merged**